### PR TITLE
Fix/rb 66 sound effect still playing

### PR DESCRIPTION
**Pull request by kw4ntiq**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [x] Norm of the code has been respected

* **In what state is your pull request?**
- [ ] Ready to merge / Waiting a reviwer to see my work.
- [x] Work In Progress (WIP) / My work is not finish but i want daily reviews. (Draft)
- [ ] CI Review / I want to see what the CI thinks of my work.
- [ ] Need feedback / Just to have feedback on what i produced. (May not be merged)

* **What kind of change does this PR introduce?** (You can choose multiple)
- [x] Bug fix
- [ ] Feature request
- [ ] New / Updated documentation
- [ ] Testing CI ( Make the pull request in draft mode)

* **What is the current behavior?** (link an issue based on the kind of change this pr introduce)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

**Review by guillaumeAbel — APPROVED** [review]

File: src/Client/Raylib/Audio/Audio.hpp [hunk 1]
```diff
@@ -10,6 +10,8 @@
 #include <string>
 #include "raylib.h"
 
+#include <iostream>
```
Comment (Saverio976) [inline src/Client/Raylib/Audio/Audio.hpp]: Logger

File: src/Client/SceneManager.cpp [hunk 2]
```diff
@@ -14,8 +14,8 @@
 
 #include "CustomTypes.hpp"
 
-constexpr int screenWidth  = 1920;
-constexpr int screenHeight = 1080;
+constexpr int screenWidth  = 800;
```
Comment (Saverio976) [inline src/Client/SceneManager.cpp]: Se rappeller de remettre 1920 180

File: src/ECS/SparseArray.hpp [hunk 3]
```diff
@@ -15,7 +15,7 @@ class SparseArray {
     public:
         void add()
         {
-            _sparse.push_back(std::size_t(-1));
+            _sparse.emplace_back(std::size_t(-1));
         }
 
         void insertBack(const Component &value)
```
Comment (Saverio976) [inline src/ECS/SparseArray.hpp]: Pas const ducoup si on std move

Comment (Saverio976) [reply src/ECS/SparseArray.hpp]: (je suis pas sûr de ça)

File: src/ECS/Systems/Systems.cpp [hunk 4]
```diff
@@ -12,6 +12,8 @@
 #include "Registry.hpp"
 #include "SystemManagersDirector.hpp"
 
+#include <iostream>
```
Comment (Saverio976) [inline src/ECS/Systems/Systems.cpp]: Logger

**Review by Saverio976 — APPROVED** [review]

**Review by guillaumeAbel — APPROVED** [review]

**Pull request merged**